### ISSUE TITLE
Home page: add links to contrast checkers ?

### ISSUE THREAD

**Issue by Vupulupu** [Vupulupu/CS312-T22-AngularPainting#13]
### /?\ Optional, but helps the graders /?\

On our home page, have a list of permalinks to the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) that ensures our site complies with WCAG (Web Content Accessibility Guidelines) AAA standard. Each permalink compares style colors that should be high-contrast, like between distinct side-by-side sections, text on backgrounds, etc.
